src/ansys/mapdl/core/mapdl_grpc.py

```diff
@@ -310,10 +330,16 @@ def __init__(
             ip = "127.0.0.1"
 
         # port and ip are needed to setup the log
-        self._port = port
+
+        if port is None:
+            from ansys.mapdl.core.launcher import MAPDL_DEFAULT_PORT
+
+            port = MAPDL_DEFAULT_PORT
+
+        self._port: int = port
```

Comment:
`port` is defined as a string|int, if you want `self._port` to be a int it should be converted if it's a string

Reply:
Done!